lib/src/model/broadcast/broadcast_analysis_controller.dart

```diff
@@ -240,6 +244,42 @@ class BroadcastAnalysisController extends _$BroadcastAnalysisController implemen
     state = AsyncData(state.requireValue.copyWith(pgnHeaders: pgnHeaders));
   }
 
+  void _handleEvalHitEvent(SocketEvent event) {
+    final path = pick(event.data, 'path').asUciPathOrThrow();
+
+    final depth = pick(event.data, 'depth').asIntOrThrow();
+    final pvs =
+        pick(event.data, 'pvs')
+            .asListOrThrow(
+              (pv) => PvData(
+                moves: pv('moves').asStringOrThrow().split(' ').toIList(),
+                cp: pv('cp').asIntOrNull(),
+                mate: pv('mate').asIntOrNull(),
+              ),
+            )
+            .toIList();
+
+    final cloudEval = CloudEval(depth: depth, position: state.requireValue.position, pvs: pvs);
+
+    _root.updateAt(path, (node) => node.eval = cloudEval);
+
+    if (state.requireValue.currentPath != path) return;
+
+    _refreshCurrentNode();
```

Comment:
should pass `true` to `shouldRecomputeRootView` here as we need  to update the move list with the new eval